react/week2/todolist/src/TodoList.js

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from "react";
+import TodoItem from "./TodoItem";
+
+function TodoList({ todos }) {
+  const [itemsArray, setItemsArray] = useState(todos);
+
+  function addNewItem() {
+    const newItem = {
+      id: Math.floor(Math.random() * 100),
+      description: "Random Text",
+    };
+    setItemsArray(itemsArray.concat(newItem));
+  }
```

Comment:
Above addNewItem is also working but have a look in another way of adding object in old array.
setItemsArray((prev) => [
      ...prev,
      {
        id: Math.floor(Math.random() * 100),
        description: "Random text
      },
    ]);